shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -96,6 +96,50 @@
 public class FlutterJNI {
   private static final String TAG = "FlutterJNI";
 
+  // BEGIN Methods related to loading for FlutterLoader.
+  /**
+   * Loads the libflutter.so library.
```

Comment:
ultra-nit: `Loads the libflutter.so Flutter engine native library.`?

Reply:
libflutter.so C++ library good?

Reply:
SG